documentation/asciidoc/computers/os/updating.adoc

```diff
@@ -26,11 +26,11 @@ Next, *upgrade* all your installed packages to their latest versions with the fo
 sudo apt full-upgrade
 ----
 
-Note that `full-upgrade` is used in preference to a simple `upgrade`, as it also picks up any dependency changes that may have been made.
+Note that `a full-upgrade` is preferred to a simple `upgrade`, as it also picks up any dependency changes that may have been made.
 
-Generally speaking, doing this regularly will keep your installation up to date for the particular major Raspberry Pi OS release you are using (e.g. Buster). It will not update from one major release to another, for example, Stretch to Buster or Buster to Bullseye.
+Generally speaking, doing this regularly will keep your installation up to date for the particular major Raspberry Pi OS release you are using (e.g., Buster). It will not update from one major release to another, for example, Stretch to Buster or Buster to Bullseye.
 
-However, there are occasional changes made in the Raspberry Pi OS image that require manual intervention, for example a newly introduced package. These are not installed with an upgrade, as this command only updates the packages you already have installed.
+However, there are occasional changes made in the Raspberry Pi OS image that require manual intervention, such as a newly introduced package. These are not installed with an upgrade, as this command only updates the packages you already have installed.
```

Comment:
Pointless change?

Reply:
I'm reading it as a writer. But, it's fine. Still on other changes.